workflow/rules/benchmarking.smk

```diff
@@ -69,9 +69,9 @@ rule chm_eval_kit:
 
 rule chromosome_map:
     input:
-        "resources/genome.fasta.fai",
+        genome_fai,
     output:
-        "resources/genome.chrmap.txt",
+        f"resources/{genome_name}.chrmap.txt",
```

Comment:
With Snakemake 7.8, there is no need here to have a wildcard for the genome name anymore. Changes to the genome will be picked up by Snakemake automatically since the params of the get_genome rule will change then.

Reply:
I see, but its not because of the workflow behavior, it's about having the reference name in the bam file.

Reply:
Ah, that makes sense.

Reply:
I wonder whether instead of having it implicitly in the filename one should not rather make full use of the SAM spec and add AS, SP, and DS tags to the header.

Reply:
For assembly, species and description.

Reply:
This would be more machine readable than a filename that people would still need to understand.